.travis.yml

```
@@ -0,0 +1,86 @@
+# .travis.yml for CollabDesk (Frontend + Backend unified CI/CD)
+dist: jammy
+os: linux
+language: python
+python:
+  - "3.12"
+
+# Run jobs in parallel (matrix build)
+jobs:
+  include:
+    # ===== Backend Job =====
+    - stage: Backend Tests & Deploy
+      name: "Backend: Django + Python 3.12"
+      before_install:
+        - cd backend/collabdesk
+        - python -m venv venv        
+        - source venv/bin/activate        
+        - python -m pip install --upgrade pip setuptools wheel
+      install:
+        - pip install -r requirements.txt
+        - pip install black flake8 coverage coveralls
+      script:
+        # Formatting and lint checks
+        - black .
```

I think we should use black . --check instead of black ..
The former performs a format check and will fail the build if the code doesn’t meet the formatting standards, while the latter automatically reformats files within the Travis build environment and passes the check regardless of the original file formatting.
Our expected behavior should be to reject commits with incorrect formatting rather than silently fixing them during CI.